lib/k8s/stack.rb

```diff
@@ -94,7 +94,7 @@ def prepare_resource(resource, base_resource: nil)
 
     # @param client [K8s::Client]
     # @return [Array<K8s::Resource>]
-    def apply(client, prune: true, patch: true)
+    def apply(client, patch: true, prune: true, owner_reference: nil)
```

Comment:
Not completely happy that we need to add yet-another parameter here but I get why it's needed.